### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by rpdome**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -129,6 +129,10 @@ You can get the binaries from Maven central repo. AAR package can be included as
 ```gradle
 repositories {
     mavenCentral()
+    maven {
+        // Surface Duo SDK dependency.
+        url 'https://pkgs.dev.azure.com/MicrosoftDeviceSDK/DuoSDK-Public/_packaging/Duo-SDK-Feed%40Local/maven/v1'
```
Comment (iambmelt) [inline README.md:134]: We might want to call out that this is preview

Comment (iambmelt) [reply README.md:134]: And that including this is required in your `build.gradle` is (or is not) required as of ADAL version x.x.x

Comment (rpdome) [reply README.md:134]: This is their public feed - and it's not just for preview. (future versions will be published here too. AFAIK there is no plan to publish them to maven central).

Comment (iambmelt) [reply README.md:134]: Good to know. Thanks @rpdome

**Pull request merged**